src/sql/magic_cmd.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import argparse
+
+from IPython.utils.process import arg_split
+from IPython.core.magic import (
+    Magics,
+    line_magic,
+    magics_class,
+)
+from IPython.core.magic_arguments import argument, magic_arguments
+from IPython.core.error import UsageError
+
+
+try:
+    from traitlets.config.configurable import Configurable
+except ImportError:
+    from IPython.config.configurable import Configurable
+
+
+from sql import inspect
+
+
+class CmdParser(argparse.ArgumentParser):
+    def exit(self, status=0, message=None):
+        if message:
+            self._print_message(message, sys.stderr)
+
+    def error(self, message):
+        raise UsageError(message)
+
+
+@magics_class
+class SqlCmdMagic(Magics, Configurable):
+    """%sqlcmd magic"""
+
+    @line_magic("sqlcmd")
```

Comment:
missing telemetry?

Reply:
telemetry is in the underlying functions called from this magic